coldfront/core/utils/management/commands/initial_setup.py

```diff
@@ -8,31 +8,39 @@
 
 
 class Command(BaseCommand):
-    help = 'Run setup script to initialize the Coldfront database'
+    help = "Run setup script to initialize the Coldfront database"
 
     def add_arguments(self, parser):
-         parser.add_argument("-f", "--force_overwrite", help="Force intial_setup script to run with no warning.", action="store_true")
+        parser.add_argument(
+            "-f",
+            "--force_overwrite",
+            help="Force intial_setup script to run with no warning.",
+            action="store_true",
+        )
 
     def handle(self, *args, **options):
-            if options['force_overwrite']:
-                run_setup()
+        if options["force_overwrite"]:
+            run_setup()
+
+        else:
+            print(
+                """WARNING: Running this command initializes the ColdFront database and may modify/delete data in your existing ColdFront database. This command is typically only run once."""
+            )
+            user_response = input("Do you want to proceed?(yes):")
 
+            if user_response == "yes":
+                run_setup()
             else:
-                print("""WARNING: Running this command initializes the ColdFront database and may modify/delete data in your existing ColdFront database. This command is typically only run once.""")
-                user_response = input("Do you want to proceed?(yes):")
-            
-                if user_response == "yes":
-                    run_setup()
-                else:
-                    print("Please enter 'yes' if you wish to run intital setup.")
+                print("Please enter 'yes' if you wish to run intital setup.")
```

Comment:
@gunnarwustl this is minor. Nonetheless, the message should be "no action taken" or something like that.

Reply:
I just realized that the message had been there from previous changes. My comment is just indicating that we need clarity. This could be done in future development.